_topic_maps/_topic_map.yml

```diff
@@ -290,6 +290,12 @@ Topics:
   Topics:
   - Name: Installing an on-premise cluster using the Assisted Installer
     File: installing-on-prem-assisted
+- Name: Installing with Agent
```

Comment:
Can we better say `Installing with Agent-based installer for OpenShift`

Reply:
I'd suggest: Installing with Agent-based installer (no need to say "for OpenShift")

Reply:
I agree with this >> Installing with Agent-based installer